plugins/dbqueuedb/dbqueuedb.c

```diff
@@ -366,15 +366,15 @@ static unsigned long long dbqueue_get_front_genid(struct dbtable *table,
     void *fnddta;
     size_t fnddtalen;
     size_t fnddtaoff;
-    const uint8_t *open;
+    const uint8_t *status;
     pthread_mutex_t *mu;
     pthread_cond_t *cond;
 
     init_fake_ireq(table->dbenv, &iq);
     iq.usedb = table;
     genid = 0;
 
-    rc = bdb_trigger_subscribe(table->handle, &cond, &mu, &open);
+    rc = bdb_trigger_subscribe(table->handle, &cond, &mu, &status);
     if (rc != 0) {
```

Comment:
Need special handling for paused here...?

Reply:
Given how this function is used, I think returning a zero genid when paused is the correct behavior, i.e. this function is only used to determine when an inactivity warning should be emitted into the log file and I believe that when a queuedb is paused, it should not have these warnings.